### ISSUE TITLE
OCADU Fixes

### ISSUE THREAD

**Issue by adorayi** [cds-snc/design-gc-conception#679]
Resolve issues to get the GC Design System documentation website to meet WCAG 2.1 AA standards. Report attached.[IDRC_GC Design System_2.1_2024 (1).docx](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBMVVYQXc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--85579567f06e8c6292fc46a05d41252ebf780ed6/IDRC_GC%20Design%20System_2.1_2024%20%281%29.docx) 
- 📝 Google Drive link: [IDRC_GC Design System_2.1_2024](https://docs.google.com/document/d/1shoMkQgOrNAI-eOUKfgU27BU_OP80h1h/edit?usp=drive_link&ouid=118176498651140651076&rtpof=true&sd=true)

- ✅ [Developer checklist](https://docs.google.com/document/d/1cUQjIqsfRqdlq_AIV9G9ZTH5orTeDFl0ckiXDFCwjzA/edit#heading=h.57q4fd2vx86p) to breakdown these tasks

**Comment by daine** [comment]
_For increased visibility_
Tickets in this epic:
- #680
- #681 
- #682 
- #683 
- #684
- #685 
- #686 
- #687 
- #688 
- #689 
- #690
- #691 
- #692 
- #693
- #694

**Comment by ClementineHahn** [comment]
double check @abdul all those tickets are *Actually* into done , as we're good at missing steps
